src/MyPlot/subcommand/UnDenySubCommand.php

```diff
@@ -39,10 +39,11 @@ public function execute(CommandSender $sender, array $args) {
 			$sender->sendMessage(TextFormat::RED . $this->translateString("undenyplayer.failure", [$dplayer]));
 			return true;
 		}
+		$dplayer = $this->getPlugin()->getServer()->getOfflinePlayer($dplayer)->getPlayer() ?? $this->getPlugin()->getServer()->getOfflinePlayer($dplayer);
```

Comment:
This line just looks and feels wrong. Don't they both return the same?

Reply:
`Server::getOfflinePlayer` returns an OfflinePlayer instance
`OfflinePlayer::getPlayer()` returns a Player instance or null if the player is not online